core/src/authentication/authentication.controller.ts

```diff
@@ -36,13 +36,21 @@ export class AuthenticationController {
     @UseGuards(GoogleAuthGuard)
     async googleLogin(@Request() req: Req, @Response() res: Res, @Query("state") state?: string): Promise<void> {
         const googleProfile = GoogleProfileSchema.safeParse(req.user);
-        if (!googleProfile.success) throw new HttpException("Could not validate user", 400);
+        if (!googleProfile.success) {
+            res.redirect(`${config.auth.frontend_callback}?status=error&message=${"Failed to parse provider profile"}`);
```

Comment:
This error message needs to be more user friendly; we can log information to give developers an idea of what went wrong (can generate some id to associate the user's message with the log message), but this doesn't tell an end user anything